dts/riscv/it8xxx2.dtsi

```diff
@@ -42,9 +43,174 @@
+					 NO_FUNC NO_FUNC  NO_FUNC 0xf016f6>;
+			func4_en_mask = <0       0        0       0
+					 0       0        0       0x80    >;
+		};
+		pinmuxd: pinmux@f01628 {
+			compatible = "ite,it8xxx2-pinmux";
+			reg = <0x00f01628 0x0008>;
+			label = "PINMUXD";
+			func3_gcr =     <NO_FUNC NO_FUNC  NO_FUNC NO_FUNC
+					 NO_FUNC 0xf016f0 NO_FUNC NO_FUNC>;
+			func3_en_mask = <0       0        0       0
+					 0       0x02     0       0      >;
+			func4_gcr =     <NO_FUNC NO_FUNC  NO_FUNC NO_FUNC
+					 NO_FUNC NO_FUNC  NO_FUNC NO_FUNC>;
+			func4_en_mask = <0       0        0       0
+					 0       0        0       0      >;
+		};
+		pinmuxe: pinmux@f01630 {
+			compatible = "ite,it8xxx2-pinmux";
+			reg = <0x00f01630 0x0008>;
+			label = "PINMUXE";
+			func3_gcr =     <0xf02032 NO_FUNC  NO_FUNC NO_FUNC
+					 NO_FUNC  0xf016f0 NO_FUNC 0xf02032>;
+			func3_en_mask = <0x01     0        0       0
+					 0        0x08     0       0x01    >;
+			func4_gcr =     <0xf016f3 NO_FUNC  NO_FUNC NO_FUNC
+					 NO_FUNC  NO_FUNC  NO_FUNC NO_FUNC >;
+			func4_en_mask = <0x01     0        0       0
+					 0        0        0       0       >;
+		};
+		pinmuxf: pinmux@f01638 {
+			compatible = "ite,it8xxx2-pinmux";
+			reg = <0x00f01638 0x0008>;
+			label = "PINMUXF";
+			func3_gcr =     <NO_FUNC NO_FUNC 0xf016f0 0xf016f0
+					 NO_FUNC NO_FUNC 0xf016f1 0xf016f1>;
+			func3_en_mask = <0       0       0x02     0x02
+					 0       0       0x10     0x10    >;
+			func4_gcr =     <NO_FUNC NO_FUNC 0xf016f1 0xf016f1
+					 NO_FUNC NO_FUNC NO_FUNC  NO_FUNC >;
+			func4_en_mask = <0       0       0x20     0x20
+					 0       0       0        0       >;
+		};
+		pinmuxg: pinmux@f01640 {
+			compatible = "ite,it8xxx2-pinmux";
+			reg = <0x00f01640 0x0008>;
+			label = "PINMUXG";
+			func3_gcr =     <0xf016f0 0xf016f0 0xf016f0 NO_FUNC
+					 NO_FUNC  NO_FUNC  0xf016f0 NO_FUNC>;
+			func3_en_mask = <0x20     0x08     0x10     0
+					 0        0        0x02     0      >;
+			func4_gcr =     <NO_FUNC  NO_FUNC  NO_FUNC  NO_FUNC
+					 NO_FUNC  NO_FUNC  NO_FUNC  NO_FUNC>;
+			func4_en_mask = <0        0        0        0
+					 0        0        0        0      >;
+		};
+		pinmuxh: pinmux@f01648 {
+			compatible = "ite,it8xxx2-pinmux";
+			reg = <0x00f01648 0x0008>;
+			label = "PINMUXH";
+			func3_gcr =     <NO_FUNC 0xf016f1 0xf016f1 NO_FUNC
+					 NO_FUNC 0xf016f5 0xf016f5 NO_FUNC>;
+			func3_en_mask = <0       0x20     0x20     0
+					 0       0x04     0x08     0      >;
+			func4_gcr =     <NO_FUNC 0xf016f5 0xf016f5 NO_FUNC
+					 NO_FUNC NO_FUNC  NO_FUNC  NO_FUNC>;
+			func4_en_mask = <0       0x04     0x08     0
+					 0       0        0        0      >;
+		};
+		pinmuxi: pinmux@f01650 {
+			compatible = "ite,it8xxx2-pinmux";
+			reg = <0x00f01650 0x0008>;
+			label = "PINMUXI";
+			func3_gcr =     <NO_FUNC NO_FUNC  NO_FUNC  NO_FUNC
+					 NO_FUNC 0xf016f0 0xf016f0 0xf016f0>;
+			func3_en_mask = <0       0        0        0
+					 0       0x04     0x04     0x04    >;
```

Comment:
GPIOI5-6: U2CTRL>1, so I think the mask needs to be 0x08.

Reply:
Yeah, it's wrong. Corrected.